docs/bot/systemd.md

```diff
@@ -32,22 +32,33 @@ Description=Draupnir Docker Container
 After=docker.service
 Requires=docker.service
 
+After=network-online.target
```

Comment:
This is a base systemd target, so I think it's a fair call.
Perhaps systemd instructions could also be provided for non-docker setups?

Reply:
>Perhaps systemd instructions could also be provided for non-docker setups?

If you aren't using docker, it's assumed that you are an advanced system admin anyways and can figure it out for yourself. We don't want to hand hold an inexperienced user into setting something up that they can't maintain (running from source is more complicated because both system packages and Draupnir's dependencies need to be up to be maintained for Draupnir and there's little chance of them doing that correctly).